tests/framework/config/cluster.go

```diff
@@ -27,4 +27,6 @@ type ClusterConfig struct {
 	PeerTLS           TLSConfig
 	ClientTLS         TLSConfig
 	QuotaBackendBytes int64
+	AuthToken         string
```

Comment:
I assume you will update some test cases to use this field in separate PR(s). Please let me know if it isn't the case.

Reply:
yeah, this field will be used in real auth tests.